uncertainties/core.py

```diff
@@ -2359,264 +658,19 @@ def __setstate__(self, data_dict):
             # instance might contain slots):
             setattr(self, name, value)
 
+ops.add_arithmetic_ops(AffineScalarFunc)
+ops.add_comparative_ops(AffineScalarFunc) 
+to_affine_scalar = AffineScalarFunc._to_affine_scalar
+
 # Nicer name, for users: isinstance(ufloat(...), UFloat) is
 # True. Also: isinstance(..., UFloat) is the test for "is this a
 # number with uncertainties from the uncertainties package?":
 UFloat = AffineScalarFunc
 
+def wrap(*args, **kwargs):
```

Comment:
`wrap` is a public function and this change loses the docstring and signature that it currently has. It would be better to keep those here.